src/settings-ui/Settings.UI/MainWindow.xaml

```diff
@@ -2,14 +2,13 @@
     x:Class="Microsoft.PowerToys.Settings.UI.MainWindow"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Views"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Views"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d"
     Closed="Window_Closed"
-    >
+    mc:Ignorable="d">
 
     <Grid>
-        <local:ShellPage/>
```

Comment:
There's probab;y a reason, but why don't we insert the content of _shellpage_ directly in here, again?

Reply:
ShellPage is of type Page.. we (might) need some of those properties, that window doesn't have.

Reply:
Curious which one that would be :)